src/testing/Home.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+test('renders correctly', () => {
+  const tree = renderer.create(<Home />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] @team great work adding test for your components, however you forgot to add test for the `Navbar` and `App` components , kindly consider adding test for them

Reply:
This test has been done well